cmd/kubeadm/app/util/version.go

```diff
@@ -53,7 +53,8 @@ var (
 func KubernetesReleaseVersion(version string) (string, error) {
 	if kubeReleaseRegex.MatchString(version) {
 		return version, nil
-	} else if kubeReleaseLabelRegex.MatchString(version) {
+	}
+	if kubeReleaseLabelRegex.MatchString(version) {
```

Comment:
Thanks for your feedback. Yeah, what you mentioned is reasonable in some cases. While not in this case, because I think since if the first `if` succeeds, the function will **directly return**, so if will never run to the second `if`.  Does it make sense?